libyara/include/yara/globals.h

```diff
@@ -57,18 +57,33 @@ extern char* yr_debug_callback_message_as_string(int message);
 
 extern char* yr_debug_error_as_string(int error);
 
-
 // Default is 0 for production, which means be silent, else verbose.
 extern uint64_t yr_debug_verbosity;
 
+extern __thread int yr_debug_indent;
```

Comment:
It looks like `__thread` is a GCC thing, I'm not sure if it is supported by Microsoft compilers.

Reply:
@plusvic Sorry about that. Just turned it into `YR_TLS` with the appropriate Microsoft definition. Is there a way to get Microsoft CI compiles working somehow to catch stuff like this?

Reply:
Ah, sorry there are Microsoft CI builds, but currently no CI builds that build with debug instrumentation, and `__thread` is currently only being used when debug instrumentation is built too.

Reply:
@plusvic any idea why the build failed? The console log seems to be cut off prematurely... :-(

Reply:
Also, it might be good to rename the jobs because the dashboard [1] turns all text after `14.0` into `...` so it's difficult to see which job is which? How to move the platform and configuration to the left of the job name in order to better differentiate the jobs on the dashboard?
```
Environment: TARGET=vs2015, APPVEYOR_BUILD_WORKER_IMAGE=Visual Studio 2015, VisualStudioVersion=14.0, platform=x86, configuration=Release, artifact_postfix=win32
Environment: TARGET=vs2015, APPVEYOR_BUILD_WORKER_IMAGE=Visual Studio 2015, VisualStudioVersion=14.0, platform=x86, configuration=Debug
Environment: TARGET=vs2015, APPVEYOR_BUILD_WORKER_IMAGE=Visual Studio 2015, VisualStudioVersion=14.0, platform=x64, configuration=Release, artifact_postfix=win64
Environment: TARGET=vs2015, APPVEYOR_BUILD_WORKER_IMAGE=Visual Studio 2015, VisualStudioVersion=14.0, platform=x64, configuration=Debug
```
[1] https://ci.appveyor.com/project/plusvic/yara/builds/36623047